hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/task/reduce/TestMerger.java

```diff
@@ -81,6 +84,8 @@ public class TestMerger {
   private File unitTestDir;
   private JobConf jobConf;
   private FileSystem fs;
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
```

Comment:
This is no longer needed, plus the import at line 75.

Reply:
done.